src/assets/js/protocol/protocol-sidemenu.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+    'use strict';
+
+    var $doc = document.documentElement;
+    var menu = document.querySelector('.mzp-c-sidemenu');
+    var menuToggle = document.querySelector('.mzp-js-toggle');
+
+    // Add class to reflect javascript availability for CSS
+    $doc.className = $doc.className.replace(/\bno-js\b/, 'js');
```

Comment:
This is may be redundant here and would collide with any other scripts doing the same thing. We should consider some kind of minimal global JS as part of protocol-core, even if all it does is set this class.

OR we can totally rethink it and not rely on this root-level script-is-working indicator. Each component can make its own check and set a class for a style hook for that component, but it sure is handy to have one hook in one place.

Reply:
I added a protocol-base.js that so far does nothing but this JS detection class but could possibly include other essentials down the road.